_config.yml

```diff
@@ -122,7 +122,7 @@ sitemap_ignore:
 versions:
   jwt_validator_java: 0.2.0
   spring_boot_starter: 0.5.0
-  okta_signin_widget: 2.6.0
+  okta_signin_widget: 2.16.0
```

Comment:
Hey!  Thanks for the update here, however at the moment `_config.yml` is not being used.  You will need to go to the each file to update the signin_widget version number.

You can use IntelliJ to search for the string `okta-signin-widget/2.14.0` to update all the required locations


This is a enhancement that we are going to be making soon to bring back this functionality. See: #48

Reply:
Thanks for reviewing @bretterer !
- update other files.
- keep changes in this config assuming it'll be used once the #48 has been solved.